### ISSUE TITLE
Strange behaviour rule: "declaration-colon-space-after"

### ISSUE THREAD

**Issue by jeroenkampinga** [stylelint/stylelint#898]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug where the rule doesn't throw an error when no space declared after colon.

> Which rule, if any, is this issue related to?

"declaration-colon-space-after" with the option "always"

> What CSS is needed to reproduce this issue?

```css
.foo {
  color:pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
   "declaration-colon-space-after": "always"
  }
}
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Throw error when no space is entered after the colon.

> What actually happened (e.g. what warnings or errors you are getting)?

No error is thrown

When 2 spaces are encountered like:

```css
.foo {
  color:  pink;
}
```

Then this error is thrown:
```stylelint: .../style.scss:24:22: Expected single space after ":" (declaration-colon-space-after)```

But it doesn't throw this error with no space.

**Comment by davidtheclark** [comment]
Fixed in 34f78809124089e6377a92b6d2296f5526b70586

**Comment by davidtheclark** [comment]
Thanks for the report!

**Comment by davidtheclark** [comment]
Fixed released in v5.
